### ISSUE TITLE
How to get no-reference indicator ?

### ISSUE THREAD

**Issue by ZNHzlq** [TingdiRen/URSCT-SESR#4]
Where is the code to get the values of UCIQE and UIQM indicators ?
Thank you !

**Comment by TingdiRen** [comment]
I'm very sorry. In trying to organize the code to post on github, I rewrote the eval file and forgot about UCIQE and UIQM, but you can easily search github for python code provided by others. In fact, I have not had much time to update the code recently as I was away on business. I hope this didn't cause you any problems.

**Comment by ZNHzlq** [comment]
@TingdiRen 
Thanks for your reply in your busy schedule. During our experiment, we found that different data division will affect the final results. Could you please provide us with your division txt?

**Comment by TingdiRen** [comment]
> @TingdiRen Thanks for your reply in your busy schedule. During our experiment, we found that different data division will affect the final results. Could you please provide us with your division txt?

I'm really sorry, I just noticed this response. However, I have already left my previous workplace where I had access to the server. When I asked today, the server I used has been formatted. Therefore, I'm sorry that I can't provide you with that partition. But in previous experiments, different partitions did not result in unexpected differences in performance metrics, so it should not significantly affect your reproducibility.
